apps/beeai-cli/src/beeai_cli/utils.py

```diff
@@ -260,3 +260,11 @@ def verbosity(verbose: bool, show_success_status: bool = True):
 def make_safe_name(resource_url: str) -> str:
     parsed = urlparse(resource_url)
     return parsed.netloc or parsed.path
+
+
+def normalize_url(url: str) -> str:
+    if not url.startswith(("http://", "https://")):
+        url = "https://" + url
+    parsed = urlparse(url)
+    # ensure no trailing slash
+    return f"{parsed.scheme}://{parsed.netloc}"
```

Comment:
Why have this as a helper instead of doing input validation? Also, why are we dropping the path and only using the host part of the URL?

Reply:
@abhi201191 We can simplify by just requiring a valid URL and validating it with Pydantic.